packages/ui/src/components/ConfirmModal/AcModal.tsx

```diff
@@ -13,6 +13,7 @@ import { ModalProps } from './type';
 const AcModal = ({
```

Comment:
I know this is old code now, but I'm wondering why do we call it `AcModal`? What does it mean?

Reply:
hmm, it's for desktop only and migrated from AC 😆